content/dokumenter/dagsordener/_index.md

```diff
@@ -0,0 +1,14 @@
++++
+template = "nyheter.html"
+title = "dagsordener" 
+description = "Dagsordener"
+aliases = ["/dagsorden"]
+slug = "dokumenter"
++++
```

Comment:
Siden dette skal være en slags seksjon for dagsordner kunne det sikkert vært fint med en egen `page_template` som er egnet for det.